pkg/deb/Makefile

```diff
@@ -53,7 +53,9 @@ horizon-cli-deb: require-version
 	cp ../../agent-install/agent-install.sh horizon-cli/usr/horizon/bin
 	cp ../../agent-install/agent-uninstall.sh horizon-cli/usr/horizon/bin
 	cp ../../agent-install/edgeNodeFiles.sh horizon-cli/usr/horizon/bin
-	cp ../../cli/bash_completion/hzn_bash_autocomplete.sh horizon-cli/etc/bash_completion.d
+	if [ -e ../../cli/bash_completion/hzn_bash_autocomplete.sh ]; then \
+          cp ../../cli/bash_completion/hzn_bash_autocomplete.sh horizon-cli/etc/bash_completion.d; \
+        fi
```

Comment:
fi does not line up with the if.

Reply:
I've fixed that, thanks for catching it.